### PR TITLE
Vcr billy poltergeist

### PR DESCRIPTION
**Pull request by decareano**
@tansaku; @zmagod, I made some changes. I used the Poltergeist driver instead of Webkit's  and I am getting better results. With Webkit and "puffing billy" I was getting tests to pass in ISOLATION which is good but I was getting log messages again. Puffing billy creates a cache of each map and I guess the logs are stubbed alongside. Just a guess. Anyway, with Poltergeist and Puffing billy I am able to get the tests to pass in ISOLATION and WITHOUT capybara log messages. I only tested our usual feature: `cucumber features/home_page/map.feature`.
Now the caveat: this setup works for me in my MAC. I cannot make it work in my VM with Ubuntu. I think zmago had it working in Linux. Please test and report results. 
https://github.com/teampoltergeist/poltergeist
https://github.com/oesmith/puffing-billy

### CODE REVIEW

File: features/billy_cache/get_maps.googleapis.com_dfce58f28fd0e45d55ef2bd2e1fe79ed13e250b7.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    #c {margin:0 auto;width:500px;text-align:left}
+    </style>
+  </head>
+  <body>
+    <h2>Sinatra doesn&rsquo;t know this ditty.</h2>
+    <img src='http://maps.googleapis.com/__sinatra__/404.png'>
+    <div id="c">
+      Try this:
+      <pre>get '/maps/api/js/AuthenticationService.Authenticate' do
+    "Hello World"
```
Comment (johnnymo87) [inline features/billy_cache/get_maps.googleapis.com_dfce58f28fd0e45d55ef2bd2e1fe79ed13e250b7.yml]: This looks like a response from our internal sinatra app.

**Comment by ZmagoD** [comment]
@decareano I get 14 errors running with or without internet connection. The problem is in the ./features/step_definitions/authentication_steps.rb:99:in `/^I have a "([^\"]+)" cookie set to "([^\"]+)"$/'
      ./features/step_definitions/authentication_steps.rb:103:in `/^cookies are approved$/'
      features/individual_charity_pages/this-is-my-org.feature:18:in `And cookies are approved'

**Comment by decareano** [comment]
@zmagod, I was just testing our main feature. I will take a look at it later.

**Comment by tansaku** [comment]
@decareano do check out what jpb and yana are doing with a simple rails app to display the do-it data.  It might be a good idea to help them get that set up with VCR and puffing billy to practice getting everything properly sandboxed for a smaller app before getting everything in LS ... see slack for details ... does it make sense to close this pull request in the meantime?

**Pull request closed without merging**